docs/evaluate/temporal-cloud/pricing.mdx

```diff
@@ -29,182 +29,408 @@ tags:
+For more exact estimates, please reach out to [our team](https://pages.temporal.io/ask-an-expert).
 
-## Action pricing {#action}
+Cost and billing information are available directly in the account.
+For more information, visit the [Billing and Cost](/cloud/billing-and-cost) page.
 
-**What is an Action?**
+## Temporal Cloud pricing model {#pricing-model}
 
-Actions serve as the primary unit of consumption-based pricing for Temporal Cloud.
-Temporal Actions are the building blocks for Workflow Executions.
-When you execute a Temporal Workflow, its Actions create the ongoing state and progress of your Temporal Application.
+This section explains the basis of the Temporal Cloud pricing model and how it works.
+Your total invoice each calendar month is the combination of Temporal Cloud consumption ([Actions](#action) and [Storage](#storage)) and [Support](#support).
 
-Actions are collected and billed monthly for each Namespace.
-The base rate is &#36;25 per one million Actions per month (across all Namespaces).
-You will be billed for a prorated portion of the Actions you use.
-If you use fewer than one million Actions per month, your bill for Actions will be less than &#36;25 for that month.
-
-| Actions | &#36;25 per one million (prorated) |
-| ------- | ---------------------------------- |
+### Actions {#action}
 
-Temporal Cloud also offers a credit system.
-Credits provide an additional discount schedule for billable Actions and [storage](#storage).
-Credits do not expire.
+**What are Temporal Actions?**
 
-The credits applied to a Temporal Cloud Account include volume-based discounts for each Namespace.
-Pricing may not be uniform per Namespace.
-For example, under one credit commitment, a Namespace may use up to 300 million Actions priced at &#36;23.25 per million Actions.
-Another Namespace with over 300 million Actions benefits from a reduced rate of &#36;18.80 per million Actions for any Actions beyond the 300 million mark.
+Actions are the primary unit of consumption-based pricing for Temporal Cloud.
+They track billable operations within the Temporal Cloud Service, such as starting Workflows, recording a Heartbeat or sending messages.
 
-The following table outlines the tiers for the credits system:
-
-| Actions (Per Million Actions per Namespace) | Cost       |
-| ------------------------------------------- | ---------- |
-| Up to 300M                                  | &#36;23.25 |
-| Over 300M up to 1.5B                        | &#36;18.80 |
-| Over 1.5B up to 7.5B                        | &#36;14.10 |
-| Over 7.5B up to 30B                         | &#36;10.50 |
-| Over 30B up to 150B                         | &#36;7.90  |
-| Over 150B                                   | &#36;5.90  |
+<details>
 
-The following operations result in Actions:
+<summary>
+[Toggle to View] The following operations result in Actions:
+</summary>
 
-**Workflows**
+**WORKFLOWS**
 
-- **Workflow started.**
+- **Workflow started**.
```

Comment:
Do we want to link Signal with Start?

Reply:
I'm going to courteously decline. Workflow Executions can be started in so many ways.

Reply:
I meant the line after this(line 64) sorry, I commented the wrong line

Reply:
Fixed!